packages/features/ee/workflows/components/WorkflowStepContainer.tsx

```diff
@@ -300,6 +299,33 @@ export default function WorkflowStepContainer(props: WorkflowStepProps) {
       value: trigger,
     };
 
+    const editorComponent = ({ isEditable }) => {
```

Comment:
```suggestion
    const EditorComponent = ({ isEditable }) => {
```
Also add proper types @Flash-Shivam